webpack.config.js

```diff
@@ -26,7 +53,7 @@ module.exports = () => {
         publicPath: config.web_app_url + '/',
       }),
     },
-    // target: 'node',
+    target: 'web', // this seems like it should be "node" or "electron-renderer" but those both crash
```

Comment:
Do you think we can just remove this line? I was getting some crashing because of it for a while, I can't reproduce now but just in case maybe we should keep this as it was. From the docs it looks like it should use the `browserslist` by default.

Reply:
https://webpack.js.org/configuration/target/

Reply:
Ah yeah I will comment it again.